app/models/gist-file.js

```diff
@@ -10,7 +11,7 @@ export default DS.Model.extend({
   /*
     Replace dots with slashes, Gists can't have directories
    */
-  filePath: Em.computed('fileName', function(key, value) {
```

Comment:
For sure. Didn't want to start making style decisions but I'm :+1: if you all are